app.rb

```diff
@@ -0,0 +1,152 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    @books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    @people.each do |person|
+      person_type = person.is_a?(Teacher) ? 'Teacher' : 'Student'
+      puts "#{person.name} (#{person_type})"
+    end
+  end
+
+  def create_person(options)
+    name = options[:name]
+    age = options[:age]
+    type = options[:type]
+    classroom = options[:classroom]
+    specialization = options[:specialization]
+    parent_permission = options.fetch(:parent_permission, true)
+
+    if type == 'teacher'
+      person = Teacher.new(name: name, age: age, parent_permission: parent_permission, specialization: specialization)
+    elsif type == 'student'
+      person = Student.new(name: name, age: age, parent_permission: parent_permission, classroom: classroom)
+    else
+      puts 'Invalid person type. Please specify either "teacher" or "student".'
```

Comment:
- [ ] @theresetuyi , Great work up to this point. However, in this milestone, you were provided with a [video guideline](https://www.youtube.com/watch?v=vkkgrhD6aXQ) on how the console app should behave. You missed some key aspects:
  - When selecting between teacher and student, the console should provide an option to use numbers to select.
  - When creating a rental, the console should present an indexed list of members that can be selected using their respective indexes

- Keep up the good work 👍